src/lib/DynamicContent.ts

```diff
@@ -233,7 +233,7 @@ export class DynamicContent {
   /**
    * @hidden
    */
-  private client: HalClient;
+  public client: HalClient;
```

Comment:
Why has this been made public? I can't see it being used anywhere.

Reply:
Good point, it's the old logic change that should be reverted. Thanks